protocols/player.go

```diff
@@ -0,0 +1,8 @@
+package protocols
+
+type Player interface {
+	ID() (*string, error)
+	LocalizedName() (*string, error)
+	FullName() (*string, error)
+	SportID() (*string, error)
+}
```

Comment:
Why are all strings optional? They were not before. And from implementation you just returning string or error